src/main/java/org/pesmypetcare/webservice/dao/communitymanager/ForumDaoImpl.java

```diff
@@ -174,6 +180,35 @@ public void deleteMessage(String parentGroup, String forumName, String creator,
         documentAdapter.commitBatch(batch);
     }
 
+    @Override
+    public void reportMessage(String parentGroup, String forumName, String creator, String reporter, String date)
+        throws DatabaseAccessException, DocumentException, InvalidOperationException {
+        if (creator.equals(reporter)) {
+            throw new InvalidOperationException("409", "A message creator can't report its message");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "409" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5633182)